<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMedicinePrescriptionDialog</class>
 <widget class="QDialog" name="AddMedicinePrescriptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thêm Thuốc</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #f5f5f5;
}
QLabel {
    font-weight: bold;
    min-width: 120px;
}
QLineEdit, QTextEdit, QSpinBox, QComboBox {
    padding: 5px;
    border: 1px solid #ccc;
    border-radius: 3px;
}
QGroupBox {
    font-weight: bold;
    border: 1px solid #CCCCCC;
    border-radius: 5px;
    margin-top: 10px;
    padding-top: 10px;
    background-color: white;
    color: black;
}
QGroupBox::title {
	font: 700 10pt &quot;Arial&quot;;
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 5px 0 5px;
    color: black;
}
QPushButton#btnSearchMedicine {
    background-color: #4CAF50;
    color: white;
    border: none;
    padding: 5px 15px;
    border-radius: 3px;
    font-weight: bold;
}
QPushButton#btnSearchMedicine:hover {
    background-color: #45A049;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>179</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>THÊM THUỐC VÀO ĐƠN</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>179</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Thông tin thuốc</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget_search" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_search">
         <item>
          <widget class="QLabel" name="label_search">
           <property name="text">
            <string>Tìm kiếm thuốc:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtSearchMedicine">
           <property name="placeholderText">
            <string>Nhập tên thuốc, tên khác hoặc danh mục...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSearchMedicine">
           <property name="text">
            <string>Tìm</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_medicine" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Tên thuốc:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbMedicine">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Liều lượng:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtDosage"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Số lần/ngày:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinFrequency"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Số ngày:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinDuration"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Hướng dẫn sử dụng:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="txtInstruction"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
